Packages/com.vrchat.UdonSharp/Editor/UdonSharpEditorManager.cs

```diff
@@ -907,7 +907,7 @@ private static IEnumerable<GameObject> GetAllPrefabsWithUdonSharpBehaviours()
                 if (prefabRoot == null)
                     continue;
                 
-                var prefabAssetType = PrefabUtility.GetPrefabAssetType(prefabRoot);
+                PrefabAssetType prefabAssetType = PrefabUtility.GetPrefabAssetType(prefabRoot);
```

Comment:
Out of curiosity - was the switch from `var` to strong types necessary for the fix, or just an additional refactor?

Reply:
Additional refactor constrained to https://github.com/vrchat-community/UdonSharp/pull/82/commits/98c565708806d692f7efe66ce5072302913318f0